src/app/routes/index.test.jsx

```diff
@@ -210,10 +211,12 @@ it('should route to and render a front page', async () => {
 it('should route to and render a media asset page', async () => {
   const pathname = '/yoruba/media-23256797';
   fetchMock.mock(`http://localhost${pathname}.json`, mediaAssetPageJson);
+  fetchMock.mock('http://localhost/yoruba/mostwatched.json', mostWatchedData);
```

Comment:
Is it also worth adding a test for a service with a variant?

Reply:
A test is added at the getMostWatchedUrl method